### ISSUE TITLE
Incorrect private key for testnet

### ISSUE THREAD

**Issue by wileyj** [stx-labs/stacks.js#1393]
Generating and importing a testnet address is currently failing on the `wif` key. 
For comparison, using the following commands on mainnet is working as expected (with correct switches for mainnnet). 

the address was created using the following:
```bash
$ cd $HOME && npm install @stacks/cli shx rimraf
$ npx @stacks/cli make_keychain -t 2>/dev/null | jq
{
  "mnemonic": "spare decade dog ghost luxury churn flat lizard inch nephew nut drop huge divert mother soccer father zebra resist later twin vocal slender detail",
  "keyInfo": {
    "privateKey": "<priv key>",
    "address": "<stx address>",
    "btcAddress": "<btc address>",
    "wif": "<wif>",
    "index": 0
  }
}
```

Importing the above bitcoin testnet address was failing with the error:
```bash
$ bitcoin-cli \
  -rpcconnect=localhost \
  -rpcport=18332 \
  -rpcuser=stacks \
  -rpcpassword=stacks \
importmulti '[{ "scriptPubKey": { "address": "<address>" }, "timestamp":"now", "keys": [ "<wif>" ]}]' '{"rescan": true}'
[
  {
    "success": false,
    "error": {
      "code": -5,
      "message": "Invalid private key encoding"
    }
  }
]
```

**Comment by wileyj** [comment]
also to note, the mnemonic is clearly fake and other values were redacted.

**Comment by janniks** [comment]
Could you try this alpha version:
`npx @stacks/cli@5.0.3-pr.0e6a8f8.0 make_keychain -t`
Maybe that PR already fixes the issues you are observing #1391

**Comment by wileyj** [comment]
I was able to generate the correct keys locally and use them to import the address/privateKey, just creating an issue so you're aware.
